create_remote_nodes.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+source common.sh
+source ocp_install_env.sh
+
+namespace=$1
+if [ $namespace == "" ]; then
+	echo "Error: Insufficient arguments. Please specify the namespace"
+	echo "Usage: $0 <namespace>"
+	exit 1
```

Comment:
We could make this default to openshift-machine-api, since there are cases (such as remote worker node testing) where this would be useful independent of the multi-namespace BMO tests?